docs/guide/security.md

```diff
@@ -151,3 +151,42 @@ security:
 ```
 
 If you need to [create your legacy SSO Handler, please read this entry](http://share.ez.no/learn/ez-publish/using-a-sso-in-ez-publish).
+
+## Security advisories
+
+!!! caution
+
+    ### Executable packages with Legacy Bridge
+
+    [This issue](http://share.ez.no/community-project/security-advisories/ezsa-2018-002-the-files-uploaded-via-packages-component-are-executable) affects installations using eZ Publish Legacy, either stand-alone, or as part of eZ Platform 5.x, or in eZ Platform 1.11 and newer using LegacyBridge. If you are not using Legacy in any way, you are not affected.
+
+    The package system, by design, allows you to package an extension into a file, and export/import such packages. Extensions can of course contain PHP scripts, and they usually do. Such scripts can be used in an attack on the server. This problem is fundamental and cannot be fixed by any other means than by removing the feature.
+
+    By default, only the Administrator has the permissions to use the package system. It follows that the Administrator role, and any others granted packaging permissions, can only be held by users who already have access to the server, and/or can be trusted not to exploit this access.
+
+    As a consequence eZ Publish legacy should not be used in the type of shared hosting installation where Administrators are not supposed to have access to the underlying operating system, or to other eZ Publish installations on the same server. The package system is an old part of eZ Publish legacy, and it was not designed for that kind of installation. Currently this is not considered best practice anyway - setups using e.g. Docker and Platform.sh allow you to completely separate installations from each other. This is a better way to keep things secure than relying on PHP scripts being read-only even for administrators. (The package system does not exist in eZ Platform and will not be added there, since extensions are not used there.)
+
+    In summary:
+
+    If you are responsible for legacy installations where administrators cannot be fully trusted not to exploit their privileges, make sure to properly lock down the package system and/or fully separate web sites from each other.
+    Make sure that the administrator password(s) are secure, and not using the default administrator password.
+
+    **Proposed quick solution for those affected:**
+
+    If you are administrating a shared hosting solution of this kind, it may take a while to change the setup. Meanwhile, one quick way to lock down the package system is to use rewrite rules to block all access to package URLs. Apache example:
+
+    `RewriteRule ^/package/.* - [R=403,L]`
+
+    or with URL-based SiteAccess:
+
+    `RewriteRule ^/my_site_access/package/.* - [R=403,L]`
+
+    or supporting both cases, and multiple SiteAccesses:
+
+    `RewriteRule ^(/my_site_access|/my_site_access_admin)?/package/.* - [R=403,L]`
```

Comment:
Wouldn't it be better with a rewrite rule that allows optional first part, something like:
`RewriteRule ^/([^/]+/)?package/.*  - [R=403,L]`